### ISSUE TITLE
Windows 10 theme

### ISSUE THREAD

**Issue by developer91234** [adirh3/Fluent-Search#960]
I know it's called fluent but it would be nice to get some legacy theme support for windows 10.

**Comment by adirh3** [comment]
You mean sharp corners?

**Comment by developer91234** [comment]
Yeah, preferably throughout the interface but at least the window itself.
